### PR TITLE
Use sticky scheduler.

### LINKED ISSUE #180: Use sticky scheduler

### PR DESCRIPTION
**Pull request by razvan**
## Description

Draft because it doesn't build yet due to transitive version conflicts.

Fixes #180 

## Review Checklist
- [x] Code contains useful comments
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] git dependencies do not reference branches.

### CODE REVIEW

File: rust/crd/Cargo.toml [hunk 1]
```diff
@@ -8,9 +8,9 @@ repository = "https://github.com/stackabletech/kafka-operator"
 version = "0.3.0-nightly"
 
 [dependencies]
-stackable-opa-crd = { git = "https://github.com/stackabletech/opa-operator.git", tag = "0.4.0" }
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.2.2" }
-stackable-zookeeper-crd = { git = "https://github.com/stackabletech/zookeeper-operator.git", tag = "0.4.0"}
+stackable-opa-crd = { git = "https://github.com/stackabletech/opa-operator.git", branch = "feature/use-sticky-scheduler" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", branch = "main" }
+stackable-zookeeper-crd = { git = "https://github.com/stackabletech/zookeeper-operator.git", branch = "feature/use-sticky-scheduler"}
```
Comment (maltesander) [inline rust/crd/Cargo.toml:13]: Switch to main branch? :)

File: rust/operator/Cargo.toml [hunk 2]
```diff
@@ -10,9 +10,9 @@ version = "0.3.0-nightly"
 [dependencies]
 stackable-kafka-crd = { path = "../crd" }
 product-config = { git = "https://github.com/stackabletech/product-config.git", tag = "0.1.0" }
-stackable-opa-crd = { git = "https://github.com/stackabletech/opa-operator.git", tag = "0.4.0" }
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.2.2" }
-stackable-zookeeper-crd = { git = "https://github.com/stackabletech/zookeeper-operator.git", tag = "0.4.0"}
+stackable-opa-crd = { git = "https://github.com/stackabletech/opa-operator.git", branch = "feature/use-sticky-scheduler" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", branch = "main" }
+stackable-zookeeper-crd = { git = "https://github.com/stackabletech/zookeeper-operator.git", branch = "feature/use-sticky-scheduler"}
```
Comment (maltesander) [inline rust/operator/Cargo.toml:15]: Switch to main branch? :)

**Review by maltesander — CHANGES_REQUESTED** [review]
Switch from feature branches to main please.

File: rust/operator/Cargo.toml (hunk 2 above)
Comment (razvan) [reply rust/operator/Cargo.toml]: Done.

File: rust/crd/Cargo.toml (hunk 1 above)
Comment (razvan) [reply rust/crd/Cargo.toml]: Done.

**Review by maltesander — APPROVED** [review]
LGTM.

**Pull request merged**